npm/src/index.ts

```diff
@@ -67,23 +68,29 @@ export const controllers = async (
   const codeStore = db.store('oauth:code', opts.db.ttl);
   const tokenStore = db.store('oauth:token', opts.db.ttl);
   const healthCheckStore = db.store('_health:check');
+  const certificateStore = db.store('saml:certificate');
```

Comment:
Should the store namespace be generic (`certificate` instead of `saml:certificate`) so that we could support any other use case in future.

Reply:
Yeah, that's a valid point.

Reply:
done